e2xgrader/preprocessors/saveautogrades.py

```diff
@@ -35,4 +46,9 @@ def _add_score(self, cell: NotebookNode, resources: ResourcesDict) -> None:
         else:
             grade.needs_manual_grade = False
 
+        if cell.metadata['nbgrader']['grade_id'] in self.cell_id.split(" "):
```

Comment:
Assuming you use a list of cell_ids above you can do:


Use this function: https://github.com/DigiKlausur/e2xgrader/blob/dev/e2xgrader/utils/nbgrader_cells.py#L9
```
# Grade this cell if either the cell_ids have not been set (len(..) == 0) or if current grade_id is in the list
if len(self.cell_ids) < 1 or grade_id(cell) in self.cell_ids:
    # Do the grading
```

Reply:
Changed the code accordingly.